content/spin/v3/community-kinesis-trigger.md

```diff
@@ -0,0 +1,131 @@
+title = "The Spin Kinesis Trigger (Community)"
+template = "spin_main"
+date = "2025-01-09T00:00:01Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/v3/community-kinesis-trigger.md"
+
+---
+
+- [Installing the AWS Kinesis Trigger](#installing-the-aws-kinesis-trigger)
+  - [Installing the AWS Kinesis Trigger Plugin](#installing-the-aws-kinesis-trigger-plugin)
+  - [Installing the AWS Kinesis Trigger Template](#installing-the-aws-kinesis-trigger-template)
+- [Creating the Application](#creating-the-application)
+  - [Inspecting the Source Code](#inspecting-the-source-code)
+- [Building and Running the Application](#building-and-running-the-application)
+- [Additional Spin Triggers](#additional-spin-triggers)
+  - [Common Triggers](#common-triggers)
+  - [Community Triggers](#community-triggers)
+
+
+Spin has experimental support for creating and running components on real time data records streamed through [AWS Kinesis](https://docs.aws.amazon.com/streams/latest/dev/introduction.html). Please note that there are only working Kinesis Trigger app samples written in [Rust](https://github.com/ogghead/spin-trigger-kinesis).
+
+> Please note: You can not `spin deploy` an application to Fermyon Cloud if it uses `kinesis` because non-HTTP triggers are not supported in Fermyon Cloud.
+
+Let's look at how the [experimental Kinesis trigger for Spin](https://github.com/ogghead/spin-trigger-kinesis) allows you to deploy an application that runs based on real time data records streamed through AWS Kinesis. (Provisioning and configuring AWS Kinesis is out of scope for this article). The Kinesis trigger leverages AWS credentials from the standard AWS configuration environment variables and maps a AWS Kinesis stream to a specific component. For example:
```

Comment:
Going into the details of credential management seems a bit out of sync with this high level.  Maybe a "configuring the Kinesis trigger" or "Kinesis trigger authentication and authorization" section further down?

Reply:
Maybe, I've almost no experience with AWS. Even the trigger repo does not provide further info

Reply:
@ogghead would you have a moment to look over this and share any thoughts on how to structure the Kinesis trigger docs?